app/v1/hateoas/IgnoreHateoasBody.scala

```diff
@@ -16,6 +16,7 @@
 
 package v1.hateoas
 
+import api.hateoas.HateoasLinks
 import config.AppConfig
```

Comment:
Same for these

Reply:
This file are not used anywhere so I have deleted it.